bin/asl-backup-menu

```diff
@@ -687,8 +690,14 @@ do_remove_local() {
     #echo ${BACKUP_ARCHIVES[@]}
 
     for BACKUP_ARCHIVE in $BACKUP_ARCHIVES; do
+	BACKUP_ARCHIVE="${BACKUP_ARCHIVE#\"}"
+	BACKUP_ARCHIVE="${BACKUP_ARCHIVE%%\"}"
```

Comment:
I see that you are removing the double quotes here that were returned by whiptail. Perhaps this could be fixed at the source command by passing `--separate-output` to the whiptail?

Reply:
We learn something new every day.  Thank you!